pymesync/pymesync.py

```diff
@@ -164,16 +164,17 @@ def update_time(self, time, uuid):
         to TimeSync.
         ``uuid`` contains the uuid for a time entry to update.
         """
-        if time['duration'] < 0:
-            return {self.error: "time object: duration cannot be negative"}
+        if 'duration' in time:
+            if time['duration'] < 0:
```

Comment:
Ah, I see you moved following statements into the first if. Disregard, deleting.